assembly/pom.xml

```diff
@@ -240,6 +240,9 @@
                                         <copy todir="target/SPLICEMACHINE-${project.version}.${envClassifier}.${parcel.patch}/bin">
                                             <fileset dir="common/src/main/resources/bin" includes="sqlshell.sh"/>
                                         </copy>
+                                        <copy todir="target/SPLICEMACHINE-${project.version}.${envClassifier}.${parcel.patch}/bin">
```

Comment:
I don't think we should include _winsqlshell.cmd_ into Cloudera parcel.

Reply:
You're right, I thought we would ship it with all supported deliverables, but this is incorrect, we will just add it to the `sqlshell` profile. Thank you for the hint.